lib/ansiblelint/utils.py

```diff
@@ -276,9 +276,13 @@ def append_children(taskhandler, basedir, k, parent_type, results):
 
 def _roles_children(basedir, k, v, parent_type, main='main'):
     results = []
+    print v
     for role in v:
         if isinstance(role, dict):
             if 'role' in role or 'name' in role:
```

Comment:
This'd probably be smth like
```suggestion
             if role.get('role', role.get('name')) is not None:
```